include/osquery/config.h

```diff
@@ -131,7 +131,7 @@ class Config : private boost::noncopyable {
    */
   void addPack(const std::string& name,
                const std::string& source,
-               const boost::property_tree::ptree& tree);
+               const rapidjson::Value& obj);
```

Comment:
We should at least document when we pass Values and when we pass objects (to establish some sort of consistency)